client/src/app/tabs/panel/tabs/LintingTab.js

```diff
@@ -72,17 +72,25 @@ function LintingIssue(props) {
   } = props;
 
   const {
+    category,
     id,
     name,
     message
   } = issue;
 
-  return <div className={ classnames(css.LintingIssue, 'linting-issue') }>
-    <div className="linting-issue__icon">
-      <ErrorIcon viewBox="2 2 20 20" />
+  return <div
+    onClick={ onClick }
+    className={ classnames(css.LintingIssue, 'linting-issue', {
+      'linting-issue--error': category === 'error',
+      'linting-issue--warning': category === 'warn'
+    }) }>
+    <div className="linting-issue__header">
+      { category === 'error' ? <ErrorIcon width="16" height="16" /> : null }
+      { category === 'warn' ? <WarningIcon width="16" height="16" /> : null }
+      <span className="linting-issue__label">{ name || id }</span>
```

Comment:
For some reason, your demo warning does not show the name, but the ID:

https://user-images.githubusercontent.com/56470362/199668465-7fe52680-8fcc-4386-81db-a9bb9823020f.mov

Reply:
This is expected behavior. Unless the rule specifies `name` as part of the report the ID will be shown. I took the rule from our standard bpmnlint rules which do not have this feature yet.